README.md

```diff
@@ -66,6 +66,8 @@ Clone the repository
 
 - While implementing a new feature or fixing an issue, there may be a chance to break the functions of `setup.py` script due to the change in the codebase (update the app name, the package directory, etc.).
 
+- When increasing the app version and build number of the template, `APP_VERSION` and `BUILD_NUMBER` variables should be updated accordingly.
```

Comment:
not sure why do we have to? 🤔  can we just keep a static version like `1.1.1 (11)` ?

Reply:
first, I want to write a test to check the current project version, but it seems to add unnecessary effort because when we update the project version, we have to update it in the script too. I decided to not write the test and this line become redundant now. Will remove this line 😓